gen/gen.go

```diff
@@ -147,10 +147,18 @@ func WithStrictTypes(b bool) OptsFunc {
 	}
 }
 
+// WithPackageDoc configures the generator to output the given text as a package doc comment.
+func WithPackageDoc(text string) OptsFunc {
+	return func(g *Generator) {
+		g.pkgdoc = text
```

Comment:
I would suggest we check here the it ends with a full stop, as this is Go convention.

Reply:
Do you mean we should add a full stop if there isn't one?

Reply:
Yes, that is the suggestion